### PR TITLE
Changing API calls to use most recent server deployment.

### PR DESCRIPTION
**Pull request by erikrramsey**
Added Api.js, used to change easily between development api call url from deployment url.

Changed these files to import Api.js and use API_URL:
* screens/Forum/ForumScreen.js
* screens/Home/GetData.js
* screens/Progress/GoalSetter.js
* screens/Quiz/QuizScreen.js
* screens/Ranking/RankingScreen.js

### CODE REVIEW

**Review by mcebaniqued — APPROVED** [review]
Everything looks good! This definitely helps not having to set up server on local host anymore.

File: Carbon/navigation/screens/Api.js
Comment (mcebaniqued) [inline Carbon/navigation/screens/Api.js:1]: Would it be better if we put this inside components folder?

Comment (erikrramsey) [reply Carbon/navigation/screens/Api.js:1]: Maybe a new config folder would make sense ?

File: Carbon/navigation/screens/Api.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Testing Server
+//export const API_URL = "http://192.168.0.150:3000/api/"
+
+// Deployed Server
+export const API_URL = "http://Carbonserver-env.eba-pmpdtmpe.us-east-1.elasticbeanstalk.com/api/"
```
Comment (SnellJ2) [inline Carbon/navigation/screens/Api.js:5]: Good work for helping us deploy the backend server, my only question is, is this a security risk having this link in the code itself?

Comment (SnellJ2) [reply Carbon/navigation/screens/Api.js:5]: If it is we should fix that before deploying to app stores.

**Review by SnellJ2 — APPROVED** [review]
Good work, appreciate you taking the initiative when it comes to the server a lot.

File: Carbon/navigation/screens/Api.js (hunk 1 above)
Comment (erikrramsey) [reply Carbon/navigation/screens/Api.js]: that's the public facing link to the server, its just a url it doesnt have any keys or credentials

**Comment by erikrramsey** [comment]
added a config folder and updated imports, going ahead and merging

**Pull request merged**